rp2040-hal/src/pll.rs

```diff
@@ -264,3 +265,28 @@ impl<D: PhaseLockedLoopDevice> PhaseLockedLoop<Locking, D> {
         self.transition(Locked)
     }
 }
+
+/// Blocking helper method to setup the PLL without going through all the steps.
+pub fn setup_pll_blocking<D: PhaseLockedLoopDevice, R: Rate>(
+    dev: D,
+    xosc_frequency: Generic<u32>,
+    config: PLLConfig<R>,
+    clocks: &mut ClocksManager,
+    resets: &mut RESETS,
+) -> Result<PhaseLockedLoop<Locked, D>, Error>
+where
+    R: Into<Hertz<u64>>,
+{
+    // Before we touch PLLs, switch sys and ref cleanly away from their aux sources.
```

Comment:
Do we need to do something with the USB or ADC clocks as well here?

Reply:
Not sure, tbh I feel this is a bit of a hack